File: expenses-manager-backend-java/src/main/java/com/github/beliakou/expenses/controller/StatusController.java
package com.github.beliakou.expenses.controller;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.ResponseStatus;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class StatusController {

    @GetMapping("/ping")
    @ResponseStatus(HttpStatus.OK)
    public void ping() {
    }
}

File: expenses-manager-backend/expenses-manager-server/src/main/kotlin/com/github/beliakou/expenses/server/controller/StatusController.kt
package com.github.beliakou.expenses.server.controller

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class StatusController {

    @RequestMapping("/ping")
    fun ping() : ResponseEntity<String> {
        return ResponseEntity.ok("hello")
    }
}